packages/flutter_app/pubspec.lock

```diff
@@ -234,7 +234,7 @@ packages:
     source: hosted
     version: "0.4.1"
   clock:
-    dependency: transitive
+    dependency: "direct main"
```

Comment:
We have direct access to the clock package so that the github ci can pass through.